app/models/concerns/netlify_content.rb

```diff
@@ -36,8 +46,12 @@ def has_field(*fields, default: nil)
     attr_reader :parsed_file
   end
 
+  def content_document
```

Comment:
[praise] really smart way to get this working both with direct output and the custom parsers